Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -833,9 +833,9 @@ WebPage::WebPage(PageIdentifier pageID, WebPageCreationParameters&& parameters)
         synchronizeCORSDisablingPatternsWithNetworkProcess();
     pageConfiguration.corsDisablingPatterns = parseAndAllowAccessToCORSDisablingPatterns(m_corsDisablingPatterns);
 
-    pageConfiguration.maskedURLSchemes = parameters.maskedURLSchemes;
-    pageConfiguration.loadsSubresources = parameters.loadsSubresources;
-    pageConfiguration.allowedNetworkHosts = parameters.allowedNetworkHosts;
+    pageConfiguration.maskedURLSchemes = WTFMove(parameters.maskedURLSchemes);
+    pageConfiguration.loadsSubresources = WTFMove(parameters.loadsSubresources);
```

Comment:
`loadsSubresources` is a Boolean. Perhaps we don't need to move this?

Reply:
Oh, that's a good point. I'll remove that bit.